src/blosc2/ndarray.py

```diff
@@ -574,6 +574,24 @@ def keep_last_read(self, value: bool):
     def info(self):
         """
         Print information about this array.
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> my_array = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+        >>> print(np.info(my_array))
```

Comment:
You are showing a NumPy example, not a Blosc2 example ;-)

Reply:
Done!